common/message_types.txt

```diff
@@ -1,15 +1,15 @@
+#
+
 message_type = {
 	key = hfe_message_type_pirate_raid
 	icon = "GFX_hfe_message_pirate_raid"
 	icon_frame = 1
 
 	name = hfe_message_type_pirate_raid_name
-	sound = hfe_advisor_notification_pirate_raid
+	sound = "notification"
```

Comment:
Had, the reason that I cut the Pirate Notification Message Types is because you changed the files so that it spawns a dynamic notification. Said dynamic notification need not replace a Vanilla rendition file and no longer requires such a drastic action to be done.

That whole file was meant to be deleted. (I also changed back the alert to "hfe_advisor_notification_pirate_raid".)

Reply:
I do not see why pirate should not have its own notification